internal/v2/command/user_human_otp_model.go

```diff
@@ -10,9 +10,9 @@ import (
 type HumanOTPWriteModel struct {
 	eventstore.WriteModel
 
-	Secret *crypto.CryptoValue
-
-	State domain.OTPState
+	MFAState domain.MFAState
+	Secret   *crypto.CryptoValue
+	OTPState domain.OTPState
```

Comment:
do we really need two states? ... i find it rather confusing

Reply:
no